flu/flu.prepare.py

```diff
@@ -79,7 +81,8 @@ def make_config(lineage, resolution, params):
             config = make_config(lineage, resolution, params)
             runner = prepare(config)
             runner.load_references()
-            runner.applyFilters()
+            if params.strains is None:
```

Comment:
I really like the ability to restrict to taxa in a text file. 
If we transfer this flag from `params` to the config file, then this can be checked inside the `applyFilters` method. I think this makes it a little simpler to roll out to other datasets - as it currently stands we'd need to add a conditional to evert `X.prepare.py`script